generator/common.ts

```diff
@@ -122,6 +122,20 @@ export const CHAIN_TO_CHAIN_ID = {
   ZkSync: zkSync.id,
 };
 
+export const CHAIN_TO_CHAIN_SCAN = {
```

Comment:
@Rozengarden maybe better to use a method like this [one](https://github.com/bgd-labs/aave-address-book/blob/4da912141dabc105105b1e7f29d35e7fb3d19f63/scripts/generator/utils.ts#L6-L13) so we don't have to maintain the list